examples/schema_template_contract/main.tf

```diff
@@ -13,17 +13,93 @@ provider "mso" {
   insecure = true
 }
 
+resource "mso_tenant" "tenant1" {
+  name         = "test_tenant"
+  display_name = "test_tenant"
+  description  = "DemoTenant"
+  site_associations {
+    site_id = mso_site.test_site.id
+  }
+}
+
+resource "mso_schema" "schema1" {
+  name          = "test_schema"
+  template_name = "Template1"
+  tenant_id     = mso_tenant.tenant1.id
+}
+
+resource "mso_schema_template_filter_entry" "filter_entry" {
+  schema_id           = mso_schema.schema1.id
+	template_name       = mso_schema.schema1.template_name
+	name                = "Filter1"
+	display_name        = "Filter1"
+	entry_name          = "entry1"
+	entry_display_name  = "entry1"
+    entry_description   = "DemoEntry"
+    ether_type          = "arp"
+    ip_protocol         = "eigrp"
+    tcp_session_rules   = ["acknowledgement"]
+	destination_from    ="unspecified"
+	destination_to      ="unspecified"
+	source_from         ="unspecified"
+	source_to           ="unspecified"
+	arp_flag            ="unspecified"
+    stateful            = false
+    match_only_fragments= false
+}
```

Comment:
Try to format all of them so they are aligned and like terraform fmt would do them.